File: OutArray/out.c
#include <stdio.h>

int main() {
    char *input[] = {"This", "is", "an", "array"};
    int max = 4;
    for (int x=0; x<=max; x++) {
        printf("Index: %d // Value: %s\n", x, input[x]);
    }
    printf("Goodbye, I'm C!\n");

}

File: OutArray/out.php
<?php
$input = array("This", "is", "an", "array");
$max = 4;
for ($x=0; $x<=$max; $x++) {
    echo sprintf("Index: %d // Value: %s\n", $x, $input[$x]);
}
echo "Goodbye, I'm PHP!\n";

